test/blame/navigation-comma-key-test

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
I'd change `navigation-comma-key-test` to `parent-test`.

Reply:
Maybe `navigation-parent-test`?

Reply:
I renamed it to `navigation-parent-test` for now. I can rename it to `parent-test` or something else if necessary.